.circleci/config.yml

```diff
@@ -28,6 +28,8 @@ jobs:
             echo 'Building...'
             echo 'Node version: '
             node --version
+            echo 'nvm ls: '
+            nvm ls
             yarn version
```

Comment:
One other thing, when I ran `yarn version` locally there was a prompt to enter a version then it set `"version": "x.y.z"` int he package.json. So maybe we should use `yarn --version` to be safe?

Reply:
It actually detects when it's not running with a tty, so it doesn't block. It's a good idea anyway, to be safe... doesn't hurt.